src/test/java/org/broadinstitute/hellbender/tools/walkers/ValidateVariantsIntegrationTest.java

```diff
@@ -160,6 +160,15 @@ public void testBadChrCount1() throws IOException {
         );
 
         spec.executeTest("test bad chr counts #1", this);
+
+        //test with no reference and no validations specified
+        IntegrationTestSpec spec2 = new IntegrationTestSpec(
```

Comment:
Could go through all of the tests and make these `final`, but I'll leave it up to you.

Reply:
Adam put these tests in and he was missing `final`s???
I did the two in the test I modified, but I don't like having my git blame all over the place if I didn't substantively modify the tests.